src/db_tables/episode.py

```diff
@@ -0,0 +1,25 @@
+from typing import TYPE_CHECKING
+
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from src.utils.database import Base
+
+# Prevent circular imports for relationships
+if TYPE_CHECKING:
+    from src.db_tables.show import Show
+    from src.db_tables.user import User
+
+
+class Episode(Base):
+    """User database table."""
+
+    __tablename__ = "episode"
+
+    id: Mapped[int] = mapped_column(primary_key=True, index=True, nullable=False)
```

Comment:
Primary key here should be a composite of tvdb show id, user and the episode tvdb id)